test/integration/plugins/virtuals.js

```diff
@@ -196,6 +196,54 @@ module.exports = function (bookshelf) {
       deepEqual(json, null);
     });
 
+    it('virtuals with paramaters work and can be getted by `Model.get` ', function () {
+      var m = new (bookshelf.Model.extend({
+        virtuals: {
+          fullName: function(param) {
+              return this.get('firstName') + ' ' + param;
+          },
+
+          fullNameDouble: function(param1,param2) {
+              return this.get('firstName') + ' ' + param1 + ' ' + param2;
+          }
+        }
+      }))({firstName: 'Joe', lastName: 'Shmoe'});
```

Comment:
all the tests in the project use the "new" paradigm so I think it's better the current code